sdk/python/tests/test_materialize.py

```diff
@@ -138,6 +138,31 @@ def prep_local_fs_and_fv() -> Iterator[Tuple[FeatureStore, FeatureView]]:
             yield fs, fv
 
 
+# Checks that both offline & online store values are as expected
+def check_offline_and_online_features(
+    fs: FeatureStore,
+    fv: FeatureView,
+    driver_id: int,
+    event_timestamp: datetime,
+    expected_value: float,
+) -> None:
+    # Check online store
+    response_dict = fs.get_online_features(
+        [f"{fv.name}:value"], [{"driver_id": driver_id}]
+    ).to_dict()
+    assert abs(response_dict[f"{fv.name}__value"][0] - expected_value) < 1e-6
+
+    # Check offline store
```

Comment:
we do have sdk/python/tests/test_historical_retrieval.py - this works but it may be more natural to cover the field mapping cases there?

Reply:
Agree